apps/s_server.c

```diff
@@ -130,6 +130,17 @@ static unsigned int psk_server_cb(SSL *ssl, const char *identity,
 
     if (s_debug)
         BIO_printf(bio_s_out, "psk_server_cb\n");
+
+    if (SSL_version(ssl) >= TLS1_3_VERSION) {
+        /*
+         * This callback is designed for use in TLSv1.2. It is possible to use
+         * a single callback for all protocol versions - but it is preferred to
+         * use a dedicated callback for TLSv1.3. For TLSv1.3 we have
+         * psk_find_session_cb.
+         */
+        return 0;
```

Comment:
I guess a bare `return 0;` does make sense here, rather than `goto out_err;` that will print some diagnostics that are unwarranted in this case.
The existing usage of that label is quite inconsistent, though, and might merit a re-think (in a separate PR).

Reply:
Yes, this really isn't an error here. It's "we don't have a suitable PSK, but please carry on".